parsers/NTESMO.py

```diff
@@ -210,8 +256,15 @@ def fetch_consumption(
     target_datetime: datetime = arrow.now().shift(hours=-DELAY).to(AUSTRALIA_TZ),
```

Comment:
I would use the new function you created here instead.
I.e. make this optional and then modify it before calling the get data function.